### ISSUE TITLE
TCG compatibility

### ISSUE THREAD

**Issue by nedmsmith** [ietf-rats-wg/draft-ietf-rats-corim#266]
Some of the changes to CDDL in this repo are incompatible with the TCG Endorsement Architecture. 

This thread maintains a list of incompatibilities and possible ways to address them.

1. reference-triple-record - The measurement-map in `reference-triple-record` no longer has multiples. The incompatibility can be addressed by:
```
reference-triple-record = [
  environment-map
  (measurement-map / [ + measurement-map])
]
```
2. endorsed-triple-record - The measurement-map in `endorsed-triple-record` no longer has multiples. The incompatibility can be addressed by:
```
endorsed-triple-record = [
  environment-map
  (measurement-map / [ + measurement-map])
]
```
3. *next-tbd*

**Comment by nedmsmith** [comment]
Note: I created the `tcg-compatible` branch to validate the cddl. See examples `comid-1a.diag` and `comid-2a.diag` .

**Comment by nedmsmith** [comment]
see PR #268

**Comment by nedmsmith** [comment]
Closed. Fixed by PR #268
